app/main.py

```diff
@@ -4,0 +1,86 @@
+import json
+from os import path
+
+from app.customer import Customer
+from app.shop import Shop
+
+
+def shop_trip() -> None:
+    _path = path.join("app", "config.json")
+    with open(_path, "r") as file:
+        data_from_file = json.load(file)
+        try:
+
+            fuel_price = data_from_file["FUEL_PRICE"]
+
+        except KeyError as e:
+            print(e, "not exist")
+
+        try:
+            customers_list = [
+                Customer.create_customer(customer)
+                for customer in data_from_file["customers"]
+            ]
+        except KeyError as e:
+            print(e, "not exist")
```

Comment:
Similar to the previous comment, the error message 'not exist' for missing keys in the JSON data is not very informative. Consider providing more context in the error message, such as which key is missing, to make debugging easier.